app/design/frontend/base/default/template/googleanalytics/ga.phtml

```diff
@@ -24,7 +24,17 @@ $_helper = $this->helper('googleanalytics');
 $_accountId = $_helper->getAccountId();
 ?>
 <?php if (!$this->helper('core/cookie')->isUserNotAllowSaveCookie()): ?>
-    <?php if($_helper->isUseUniversalAnalytics()): ?>
+    <?php if ($_helper->isUseAnalytics4()): ?>
+        <!-- BEGIN GOOGLE ANALYTICS 4 CODE -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=<?= $_accountId ?>"></script>
+        <script>
+            window.dataLayer = window.dataLayer || [];
```

Comment:
Is there any reason why Mage_GoogleAnalytics_Block_Ga::_getPageTrackingCode() wasn't used here but the code was inserted directly into ga.phtml? I missed this detail in the review

Reply:
I think I've used the same coding style that was already there from the previous implementations, If I recall correctly

Reply:
It's a small detail, but I mean to wrap this part inside _getPageTrackingCode() like for universal. 
```
gtag('js', new Date());
gtag('config', '<?= $_accountId ?>');
```
Then there's also getPageName that seems deprecated. I will open a PR to discuss it.